engine/src/flutter/shell/platform/darwin/ios/framework/Source/FlutterView.mm

```diff
@@ -78,16 +78,6 @@ - (instancetype)initWithDelegate:(id<FlutterViewEngineDelegate>)delegate
   return self;
 }
 
-static void PrintWideGamutWarningOnce() {
```

Comment:
I removed this because its just going to nag folks without wide gamut capable devices, now that it is on by default